data/json/items/armor/belts.json

```diff
@@ -307,15 +307,17 @@
     "pocket_data": [
       {
         "holster": true,
+        "volume_encumber_modifier": 1,
         "max_contains_volume": "1500 ml",
         "max_contains_weight": "1000 g",
         "max_item_length": "70 cm",
+        "description": "Clipped directly to belt",
         "moves": 60,
         "flag_restriction": [ "BELT_CLIP" ]
       }
     ],
-    "use_action": { "type": "holster", "holster_prompt": "Store tool or blade", "holster_msg": "You put your %1$s in your %2$s" },
+    "use_action": [ { "type": "holster", "holster_prompt": "Store tool or blade", "holster_msg": "You put your %1$s in your %2$s" }, { "type": "attach_molle", "size": 4 }, { "type": "detach_molle" } ],
     "flags": [ "WAIST", "WATER_FRIENDLY" ],
-    "armor": [ { "coverage": 5, "covers": [ "torso" ] } ]
+    "armor": [ { "coverage": 5, "covers": [ "torso" ], "volume_encumber_modifier": 0.7 } ]
```

Comment:
```suggestion
    "armor": [ { "coverage": 5, "covers": [ "torso" ], "volume_encumber_modifier": 0.35 } ]
```

Reply:
most these pouches on there own have a modifier of .3 so the waist belt should probably have at least close to that.